extensions/cocostudio/loader/parsers/timelineParser-2.x.js

```diff
@@ -469,7 +469,7 @@
 
         this.widgetAttributes(widget, json);
 
-        var clipEnabled = json["ClipAble"];
+        var clipEnabled = json["ClipAble"] || false;
```

Comment:
The default value is false ? Isn't the ScrollView should clip its content by default